src/script/ceph-backport.sh

```diff
@@ -228,32 +233,29 @@ function cherry_pick_phase {
             false
         fi
     fi
+    base_sha="$(printf '%s' "${remote_api_output}" | jq -r .base.sha)"
+    head_sha="$(printf '%s' "${remote_api_output}" | jq -r .head.sha)"
     merged=$(echo "${remote_api_output}" | jq -r '.merged')
     if [ "$merged" = "true" ] ; then
-        true
+        # Use the merge commit in case the branch HEAD changes after merge.
+        merge_commit_sha=$(printf '%s' "$remote_api_output" | jq -r '.merge_commit_sha')
+        if [ -z "$merge_commit_sha" ]; then
+            error "Could not determine the merge commit of ${original_pr_url}"
+            bail_out_github_api "$remote_api_output"
+            false
+        fi
+        cherry_pick_sha="${merge_commit_sha}^..${merge_commit_sha}^2"
```

Comment:
This script was switched to individual cherry picks in https://github.com/ceph/ceph/pull/30097.  Probably don't want to go back without a proof that the problem that folks were running into no longer exists.

Reply:
if that problem was caused by the `..` range syntax, you could list all of the commits in a single invocation like `git cherry-pick -x <commit1> <commit2> [...]` and still retain the ability the `git cherry-pick --continue`

Reply:
i have personally found the inability to `git cherry-pick --continue` to be a pain point of this script, so i support this change. the need to copy/paste the remaining commits manually is both tedious and error-prone. the presence of conflicts alone should be enough cause to reevaluate the backport, but bug fixes often require working through the conflicts anyway

Reply:
It sounds like this is a historical issue that I believe no longer exists. Can we merge this please (with approval)?

Reply:
I've been using this PR myself for a few dozen backports without issue.

Reply:
cc @vshankar @idryomov @cbodley

Reply:
I didn't mean my comment and this conversation to be a blocker.  Resolving and merging based on Casey's approval.